src/utils.ts

```diff
@@ -0,0 +1,24 @@
+import { readFileSync } from 'fs';
+import { resolve } from 'path';
+
+/**
+ * Checks if the argument is URL or not
+ * @param {String} path Path to document
+ * @returns {Boolean} Whether the given path is URL or not
+ */
+export function isUrl(path: string): boolean {
+  // handled cases: path that starts with http/https
+  // unhandled cases: ex. www.samerandomwebsite.com
+
+  return path.startsWith('http');
```

Comment:
I think this is fine as parser uses `node-fetch` that accepts only absolute urls